### DISCUSSION TITLE
sandbox-Karl-June13/v10/landing/wc_welcome

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#93] [Q&A]
# sandbox-Karl-June13/v10/landing/wc_welcome

HCL Commerce Cloud is a high-availability, highly scalable and customizable e-commerce platform.

http://10.190.67.78/sandbox-Karl-June13/v10/landing/wc_welcome.html

**Comment by ChristineNiro** [comment]
Test here as well
